SimCalorimetry/HcalZeroSuppressionProducers/python/hcalDigisRealistic_cfi.py

```diff
@@ -24,8 +24,8 @@
 from Configuration.Eras.Modifier_phase2_hcal_cff import phase2_hcal
 phase2_hcal.toModify( simHcalDigis,
                              useConfigZSvalues = cms.int32(1),
-                             HBlevel = cms.int32(16),
-                             HElevel = cms.int32(16),
-                             HOlevel = cms.int32(16),
-                             HFlevel = cms.int32(16)
+                             HBlevel = cms.int32(8),
```

Comment:
Hi @abdoulline - it looks like everything except for the HElevel in the call to toModify can be removed. Is that correct?